src/pymor/core/pickle.py

```diff
@@ -57,9 +57,12 @@ def loads(str):
         return unpickler.load()
 
 else:
-    from functools import partial
-    dump = partial(pickle.dump, protocol=protocol or PROTOCOL)
-    dumps = partial(pickle.dumps, protocol=protocol or PROTOCOL)
+    def dump(obj, file, protocol=None):
+        pickle.dump(obj, file, protocol=protocol or PROTOCOL)
+
+    def dumps(obj, protocol=None):
+        dumps(obj, protocol=protocol or PROTOCOL)
```

Comment:
This would infinitely recurse, no?

Reply:
True, thx for the hint. Should be fixed now ...